cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaConnectCluster.java

```diff
@@ -127,6 +165,44 @@ public Service generateService() {
                 Collections.singletonList(createServicePort(REST_API_PORT_NAME, REST_API_PORT, REST_API_PORT, "TCP")));
```

Comment:
The PR from @stanlyDoge added the metrics port also to the services. I think you should do the same.